<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditerRessources</class>
 <widget class="QDialog" name="EditerRessources">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="lay_haut">
     <item>
      <widget class="QPushButton" name="ajouter">
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="select_ressource"/>
     </item>
     <item>
      <widget class="QPushButton" name="supprimer">
       <property name="text">
        <string>Supprimer la ressource actuelle</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lay_bas">
     <item>
      <widget class="QPushButton" name="image">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="partie_ressource">
       <item>
        <layout class="QHBoxLayout" name="lay_categorie">
         <item>
          <widget class="QLabel" name="text_categorie">
           <property name="text">
            <string>Cat√©gorie</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="categorie"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="lay_pods">
         <item>
          <widget class="QLabel" name="text_pods">
           <property name="text">
            <string>Pods</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="pods"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="lay_niveau">
         <item>
          <widget class="QLabel" name="text_niveau">
           <property name="text">
            <string>Niveau</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="niveau"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="text_description">
         <property name="text">
          <string>Description :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="description"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="partie_equipement" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="text_vie_min">
            <property name="text">
             <string>Vie min</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="vie_min"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="lay_vie_max">
          <item>
           <widget class="QLabel" name="text_vie_max">
            <property name="text">
             <string>Vie max</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="vie_max"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="lay_force_min">
          <item>
           <widget class="QLabel" name="text_force_min">
            <property name="text">
             <string>Force min</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="force_min"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="lay_force_max">
          <item>
           <widget class="QLabel" name="text_force_max">
            <property name="text">
             <string>Force max</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="force_max"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="lay_PC_min">
          <item>
           <widget class="QLabel" name="text_PC_min">
            <property name="text">
             <string>PC min</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="PC_min"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="lay_PC_max">
          <item>
           <widget class="QLabel" name="text_PC_max">
            <property name="text">
             <string>PC max</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="PC_max"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="sort"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditerRessources</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditerRessources</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
